plugins/intel-usb4/README.md

```diff
@@ -0,0 +1,32 @@
+# Intel USB4
+
+## Introduction
+
+USB-4 is a hardware interface specification. This plugin supports the Intel Goshen Ridge hardware.
```

Comment:
I think you should mention that this can flash over multiple interfaces, but this is the XHCI interface.